structures/AF-P68133-F1-model_v2.simoe.2.tsv

```diff
@@ -1,377 +1,377 @@
-1	55

```

Comment:
Would it be worth setting a seed in R so the simulated o/e doesn't change on each commit?

Reply:
Definitely, will do